.travis.yml

```diff
@@ -51,21 +52,11 @@ matrix:
 
     # OSX {
     - os: osx
-      env: PROJECT_DIR=examples/bison TOOLCHAIN=default
```

Comment:
Do not remove but comment them out. It will be easeir to enable them back.

Reply:
OK I think the issue here is that I merged in `pkg.template` too early. Let me just undo that for now.